tests/xharness/Jenkins.cs

```diff
@@ -2096,7 +2096,7 @@ <li>Toggle visibility
 									writer.WriteLine ($"Build duration: {runTest.BuildTask.Duration} <br />");
 								}
 								if (test.Duration.Ticks > 0)
-									writer.WriteLine ($"Run duration: {test.Duration} <br />");
+									writer.WriteLine ($"Time Elapsed: {test.Duration + test.WaitingDuration} <br />"); 									
```

Comment:
Not entirely sure if this solves it. @mandel-macaque @chamons ?

Reply:
Since you have both values, what about Time Elapsed: Foo (waiting time x, running time y)